builder/vsphere/driver/datastore_mock.go

```diff
@@ -6,7 +6,11 @@ import (
 )
 
 type DatastoreMock struct {
-	FileExistsCalled    bool
+	FileExistsCalled bool
+	FileExistsReturn bool
+
```

Comment:
If there is no specific reason for the extra lines here I would remove them so that gofmt could do its thing and format the struct fields nicely.

Reply:
I think the pattern is to help you mentally separate which fields are used in which mock functions. I find it visually useful, but if it's really grating to you I can definitely remove the lines :D